frontend/src/App.tsx

```diff
@@ -105,11 +105,9 @@ const AppRouter = () => {
       <Route
         path="locations"
         element={
-          <ProtectedRoute>
-            <Home
-              authContext={{ onLogin, onRegister, passwordReset, authenticationState, ...rest }}
-            />
-          </ProtectedRoute>
+          <Home
```

Comment:
Why did you remove `ProtectedRoute`?

Reply:
Because if it is there it will always redirect to home when refreshed.

Reply:
But, are you sure only authenticated users are allowed to visit the homepage?